utils/travis/docker_run.sh

```diff
@@ -174,8 +174,14 @@ else
         fi
     }
 
+    checkindent() {
+            make -C data/tools reindent &&
+            git diff-index --quiet HEAD
```

Comment:
These lines are indented by 8 spaces, but line 184 (and 183) are indented by 4 spaces. Trivial, but given the subject of this PR it's something that should be fixed.

Reply:
Updated.